class-wc-calypso-bridge-dotcom-features.php

```diff
@@ -69,19 +108,46 @@ class WC_Calypso_Bridge_DotCom_Features {
 	protected static $has_ecommerce_features = null;
 
 	/**
-	 * Is Ecommerce plan.
+	 * Is a paid Ecommerce plan.
 	 *
 	 * @var bool
 	 */
 	protected static $is_ecommerce_plan = null;
 
+	/**
+	 * Is an Ecommerce plan from dotCom.
```

Comment:
Again `dotCom` => `WordPress.com`
```suggestion
	 * Is an Ecommerce plan from WordPress.com.
```